package.json

```diff
@@ -43,11 +43,11 @@
     "esvalid": "1.1.0",
     "fs-readdir-recursive": "0.1.0",
     "jshint": "2.5.11",
-    "lodash": "2.4.1",
+    "lodash": "3.0.0",
```

Comment:
Rock, I do the same lowering of ranges once bit too. Though since lodash hasn't bitten you yet (also let me know if it ever does) you might as well elevate it's dep range for the benefits mentioned above.